Northstar.CustomServers/mod/scripts/vscripts/evac/_evac.gnut

```diff
@@ -507,6 +542,15 @@ void function CheckIfAnyPlayerLeft( int evacTeam )
 			SetTeamActiveObjective( evacTeam, "EG_DropshipExtractEvacPlayersKilled" )
 			SetTeamActiveObjective( GetOtherTeam( evacTeam ), "EG_StopExtractEvacPlayersKilled" )
 			thread EvacEpilogueCompleted( null )
+
+			// score for killing the entire evacing team
+			foreach ( entity player in GetPlayerArray() )
+			{
+				if ( player.GetTeam() == evacTeam )
+					continue
```

Comment:
```suggestion
			foreach ( entity player in GetPlayerArrayOfTeam( GetOtherTeam( evacTeam ) ) )
			{
```
Small optimization however this assumes that evacs only happen in gamemodes with only MILITIA or IMC teams

Reply:
Yeah I want to avoid `GetOtherTeam` wherever possible because i want more multi-team modes